stackprof-webnav.gemspec

```diff
@@ -21,12 +21,12 @@ Gem::Specification.new do |spec|
   spec.bindir = 'bin'
   spec.executables << 'stackprof-webnav'
 
-  spec.add_dependency "sinatra", "~> 2.0.7"
-  spec.add_dependency "haml", "~> 5.1.2"
+  spec.add_dependency "sinatra", ">= 2.0.7"
+  spec.add_dependency "haml", ">= 5.1.2"
   spec.add_dependency "stackprof", ">= 0.2.13"
-  spec.add_dependency "better_errors", "~> 1.1.0"
-  spec.add_dependency "ruby-graphviz", "~> 1.2.4"
-  spec.add_dependency "sinatra-contrib", "~> 2.0.5"
+  spec.add_dependency "better_errors", ">= 1.1.0"
+  spec.add_dependency "ruby-graphviz", ">= 1.2.4"
+  spec.add_dependency "sinatra-contrib", ">= 2.0.5"
```

Comment:
`>=` is dangerous because we just assume any future versions of these gems will work fine with this project, which might not be the case. This is why i always opt for `~>`. Maybe just upgrade some deps?

Reply:
Hmm...  

I was trying to use this gem in my app's `Gemfile`. (my app uses sinatra 2.1.0)
Is this a usage you don't recommend?

If so, I'll close this patch

Reply:
You don't have to include stackprof-webnav in your app gemfile. This gem is completely decoupled from the app that generates the dumps. So you are trying to solve a problem that does not have to exist in the first place

Reply:
I see. thank you!